### PR TITLE
Check the commit messages are properly formatted

### PR DESCRIPTION
**Pull request by smortex**
Quick stats while working on this, on all commits of the master branch (excluding merge commits):

|                       | Commits | Percent |
|-----------------------|--------:|--------:|
| **Total**             |    8313 | 100.00% |
| **No problem**        |    5763 |  69.32% |
| **No signed-off-by**  |    1673 |  20.12% |
| **Malformed subject** |    1888 |  22.71% |

These stats where generated using:

```sh-session
romain@zappy ~/Projects/syslog-ng % ./tests/commits/check.sh --root |& grep FAIL | wc -l
2550
```

Fixes: #2800

### CODE REVIEW

**Comment by kira-syslogng** [comment]
This user does not have permission to start the build. Can one of the admins verify this patch and start the build?
(admin: you have the next options (make sure you checked the code):
"ok to test" to accept this pull request (and further changes) for testing
"test this please" for a one time test run
"add to whitelist" add author of a Pull Request to whitelist (globally, be careful, it means this user can trigger kira for any PR)
do nothing -> CI won't start)

**Comment by kira-syslogng** [comment]
This user does not have permission to start the build. Can one of the admins verify this patch and start the build?
(admin: you have the next options (make sure you checked the code):
"ok to test" to accept this pull request (and further changes) for testing
"test this please" for a one time test run
"add to whitelist" add author of a Pull Request to whitelist (globally, be careful, it means this user can trigger kira for any PR)
do nothing -> CI won't start)

**Comment by kira-syslogng** [comment]
This user does not have permission to start the build. Can one of the admins verify this patch and start the build?
(admin: you have the next options (make sure you checked the code):
"ok to test" to accept this pull request (and further changes) for testing
"test this please" for a one time test run
"add to whitelist" add author of a Pull Request to whitelist (globally, be careful, it means this user can trigger kira for any PR)
do nothing -> CI won't start)

**Comment by smortex** [comment]
Travis-CI seems to have connectivity issues with https://repo1.maven.org.  I had to restart the builds many times [before the test suite passed on the very same commit in my fork of the repository](https://travis-ci.com/smortex/syslog-ng/builds/116893539).

**Comment by Kokan** [comment]
@kira-syslogng ok to test

File: tests/commits/check.sh [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+
+commit_range=master..HEAD
```
Comment (Kokan) [inline tests/commits/check.sh]: Please use `origin/master`.

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Pull request merged**